### ISSUE TITLE
Threshold metadata coloring

### ISSUE THREAD

**Issue by sjanssen2** [biocore/empress#209]
I very much like the ability to color sub-trees according to their uniqueness regarding metadata categories. I think it would be even more useful if I could somewhere define a minimal number of reads (or better: percentage of relative abundance) that the combined features in that subtree should have to be colored. 
Thus, I could ignore features with only a handful of reads and focus on those that might have a biological impact.

**Comment by antgonza** [comment]
Agreed! In fact, I was thinking that it would be really cool if it would be possible to actually color or change line width based on relative abundance. The specific use case I had in mind: a longitudinal dataset that changes the abundance of a specific group over time; this could be super cool in an animation! Tentative dataset: diet or probiotics.

**Comment by ElDeveloper** [comment]
This makes sense, would be together with tree collapsing features, right?

**Comment by antgonza** [comment]
IMOO, I guess it will depend on the user, right? In some cases, the user would like to see it all genera (collapsed) and sometimes all species/strands (tips) ...

**Comment by ElDeveloper** [comment]
Gotcha, so this is in general a threshold of how many tips constitute a clade's color based on feature metadata.
